### PR TITLE
Draft: Addition of module summary texts

### PR DESCRIPTION
**Pull request by AndrewB9lab**
Documentation content update for multiple modules in the /cosmos/sdk-tutorials/academy project folder.

This PR primarily adds page content summaries to replace the "Next up" sections at the end of module pages, making each page self-contained and without reference to the presumed following page. There are also a small number of typo fixes to preexisting content.

### Change scope

The changes in this Pull Request include (please tick all that apply):

* [X] Small language/grammar fixes
* [ ] Small content fixes 
* [X] Addition of new content
* [ ] Sample code/command updates
* [ ] Platform fixes
* [ ] Other

### CODE REVIEW

**Review by rob-Hitchens — COMMENTED** [review]
Outstanding work, Andrew. Minor comments for you.

File: academy/2-main-concepts/bridges.md [hunk 1]
```diff
@@ -103,6 +103,16 @@ Do you want to dive deeper when it comes to bridges? Here are some helpful resou
...
 
-## Next up
+<HighlightBox type="synopsis">
 
-You will discover more on running a node in [Run Your Own Cosmos Chain](../3-my-own-chain/index.md).
+To summarize, this section has explored:
+
+* The Gravity Bridge, which connects Cosmos with Ethereum and allows the transfer of ERC-20 tokens to and from Cosmos-based blockchains.
+* The component elements of the Gravity Bridge and how they reliably monitor and report to Cosmos applications about events on the Ethereum blockchain, providing the necessary confidence to perform transactions.
+* The necessity of maintaining an up-to-date validator set to ensure that events coming from the Ethereum blockchain are only from reputable sources.
```
[the remark this answers is not in the document]
Comment (AndrewB9lab) [reply academy/2-main-concepts/bridges.md:112]: So, perhaps:
**The necessity of the validator set remaining up-to-date regarding activity on the Ethereum blockchain, to ensure quorum consensus is only formed on events issuing from reputable sources.**
?

File: academy/3-my-own-chain/events.md [hunk 2]
```diff
@@ -370,6 +370,19 @@ r*r*r*r*
 
 This confirms that the _play_ event is emitted as expected. You can do the same for the _game created_ event.
 
-## Next up
+<HighlightBox type="synopsis">
 
-Time to add a third message to make it possible for a player to [reject a game](./reject-game.md) and to make your checkers blockchain more resistant to spam.
+To summarize, this section has explored:
+
+* How to define event types and then emit events to notify players of game actions as they occur, such as creating games and playing moves.
```
[the remark this answers is not in the document]
Comment (AndrewB9lab) [reply academy/3-my-own-chain/events.md:377]: So, perhaps:
_How to define event types and then emit events **to cause the UI to notify** players of..._
?

File: academy/2-main-concepts/bridges.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (rob-Hitchens) [reply academy/2-main-concepts/bridges.md:112]: What about this? Possibly I'm just misunderstanding what you're saying? 

- How the validator set forms a consensus about happenings on Ethereum

File: academy/3-my-own-chain/events.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (rob-Hitchens) [reply academy/3-my-own-chain/events.md:377]: sounds good

**Comment by AndrewB9lab** [comment]
Okay, I've implemented everything from @rob-Hitchens' comments. Unless there's more you'd like to do, I think this task is wrapped from my end.

**Pull request merged**